docs/layout_examples.org

```diff
@@ -7,9 +7,89 @@
 Find below a series of examples of how you might setup your layouts or partials
 to achieve specific effect.
 
+* Blog entries and list of updates
+
+Here are two snippets that, combined, create blog posts and an ordered
+list of entries on the blog.  It is possible to create multiple
+different themed “blogs” as part of a given site, by creating multiple
+modified versions of this template.
+
+This layout formats the list of blog entries:
+#+begin_src clojure
+(defn blog
+  [{:keys [build-url title render partials]}]
+  (let [{:keys [head]} partials]
+    [:html
+     (head build-url)
+     [:body
+      [:main
+       [:article.def-wrapper
+        [:div.def-content
+         [:h1 title]
+         ;; List files that are "FIRN_UNDER" the "Updates" page
+         (render :sitemap {:start-at ["Updates"]
+                           :depth 1
+                           :sort-by :newest})
+         [:hr]
+         [:div.back-to-main
+          [:p "Back to main index: "
+           [:a {:href "./index"} "Index"]]]
+         ]]]]]))
+#+end_src
+
+The layout should be accompanied by an Org mode file like the
+following:
+
+#+begin_src org
+#+title: Updates
+#+FIRN_LAYOUT: blog
+
+(Text written here will be overwritten when the page is rendered.)
+#+end_src
+
+The following layout formats an individual blog entry that will be
+added to the list described above.
+
+#+begin_src clojure
+(defn update
```

Comment:
only one small suggestion would be to change this function name to `blog-update` or something of the like as `update` is a core clojure func and may be confusing at first glance.

Reply:
Will do!